### PR TITLE
Add 1.23.4 changelog

### PR DESCRIPTION
**Pull request by noramullen1**

### CODE REVIEW

File: content/telegraf/v1.23/release-notes-changelog.md [hunk 1]
```diff
@@ -10,6 +10,57 @@ menu:
     name: Release notes
     weight: 60
 ---
+## v1.23.3 [2022-08-17]
```
Comment (sjwang90) [inline content/telegraf/v1.23/release-notes-changelog.md:13]: Should be `v1.23.4` and could we change the date to [2022-08-16] to match the GitHub changelog?

File: content/telegraf/v1.23/release-notes-changelog.md [hunk 2]
```diff
@@ -10,6 +10,57 @@ menu:
...
+
+- Bump `github.com/lxc/lxd` to be able to run tests.
+- Sync sql output and input build constraints to handle loong64 in go1.19.
+- Update credentials file to not use `endpoint_url` parameter
```
Comment (kelseiv) [inline content/telegraf/v1.23/release-notes-changelog.md:17]:
```suggestion
- Update credentials file to not use `endpoint_url` parameter.
```

File: content/telegraf/v1.23/release-notes-changelog.md [hunk 3]
```diff
@@ -10,6 +10,57 @@ menu:
...
+- Fixes to linter issues
```
Comment (kelseiv) [inline content/telegraf/v1.23/release-notes-changelog.md:18]:
```suggestion
- Fix linter issues.
```

File: content/telegraf/v1.23/release-notes-changelog.md [hunk 4]
```diff
@@ -10,6 +10,57 @@ menu:
...
+- Cloudwatch (`cloudwatch`): Customizable batch size when querying.
+- Kube Inventory (`kube_inventory`): Send file location to enable token auto-refresh.
+- Kubernetes (`kubernetes`): Refresh token from file at each read.
+- MongoDB (`mongodb`): Update version.
+- OPC UA (`opcua`): Return an error with mismatched types.
+- SQL server (`sqlserver`): Set lower deadlock priority.
+- Stackdriver (`stackdriver`) Handle when no buckets available.
+
+
+## Dependency Updates
```
Comment (kelseiv) [inline content/telegraf/v1.23/release-notes-changelog.md:32]:
```suggestion
## Dependency updates
```

File: content/telegraf/v1.23/release-notes-changelog.md [hunk 5]
```diff
@@ -10,6 +10,57 @@ menu:
...
+- Update credentials file to not use `endpoint_url` parameter
+- Fixes to linter issues
+- Add Coralogix dialect to open telemetry.
+
+## Input plugin updates
+
+- Cloudwatch (`cloudwatch`): Customizable batch size when querying.
+- Kube Inventory (`kube_inventory`): Send file location to enable token auto-refresh.
+- Kubernetes (`kubernetes`): Refresh token from file at each read.
+- MongoDB (`mongodb`): Update version.
```
Comment (kelseiv) [inline content/telegraf/v1.23/release-notes-changelog.md:26]: should we specify version, or say latest version?

**Review by kelseiv — APPROVED** [review]

**Pull request merged**